aspnetcore/security/authorization/simple.md

```diff
@@ -73,13 +73,26 @@ This would allow only authenticated users to the `AccountController`, except for
 
 <a name="aarp"></a>
 
-## Authorize attribute and Razor Pages
+## Razor Pages authorization
 
-The <xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute> can ***not*** be applied to Razor Page handlers. For example, `[Authorize]` can't be applied to `OnGet`, `OnPost`, or any other page handler. Consider using an ASP.NET Core MVC controller for pages with different authorization requirements for different handlers.
+You can require authentication on a Razor Page using the `<xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute>` attribute:
 
-The following two approaches can be used to apply authorization to Razor Page handler methods:
+```csharp
+[Authorize]
+public class AdminModel : PageModel
+{
+    public void OnGet()
+    {
+    }
+}
+```
+
+### Razor Pages handler method authorization
+
+The <xref:Microsoft.AspNetCore.Authorization.AuthorizeAttribute> must be applied on the page instead of individual Razor Page handlers. For example, `[Authorize]` can't be applied to `OnGet`, `OnPost`, or any other page handler. The following approaches can be used if needed:
 
 * Use separate pages for page handlers requiring different authorization. Move shared content into one or more [partial views](xref:mvc/views/partial). When possible, this is the recommended approach.
+* Consider using an ASP.NET Core MVC controller for pages with different authorization requirements for different handlers.
 * For content that must share a common page, write a filter that performs authorization as part of [IAsyncPageFilter.OnPageHandlerSelectionAsync](xref:Microsoft.AspNetCore.Mvc.Filters.IAsyncPageFilter.OnPageHandlerSelectionAsync%2A). The [PageHandlerAuth](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/simple/samples/3.1/PageHandlerAuth) GitHub project demonstrates this approach:
```

Comment:
This filter workaround is, in my opinion, rather advanced. Given this is the "Simple authorization" article, I would suggest moving this workaround to a different page.

Reply:
Everyone knows you can apply to a RP, this obscures the problem. Feel free to change it to something like

`[Authorize]` must be applied to the Razor Page, it cannot be applied to individual Razor Page handlers.

The best workaround is to use a MVC controller

Reply:
> Everyone knows you can apply to a RP, this obscures the problem. 

This is the "simple" guide to adding authorization to your app. It's the first thing [the overview](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/introduction?view=aspnetcore-5.0) links to. Imagine I'm a newcomer, how do I know to add the `[Authorize]` attribute to Razor Pages already?

These docs are overwhelming with advanced details. It's good for folks to know they can't add granular authorization on Razor Pages, sure, but the filter example in particular is very low-level for an introductory guide.

Reply:
> This filter workaround is advanced. Given this is the "Simple authorization" article, I would suggest moving this workaround to a different page.

We have a long history with this article you're not familiar with. `Simple/Simply` are banned words in the repo, but I can't substitute the normal replacement, basic, because that's creates an ambiguity with [Basic Authentication](https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/authentication/basicauthentication)

I need to get ride of simple in the title and TOC, but that's another day.

We had alot of problem with folks trying to use different auth on RP handlers. This totally solved the problem.